src/components/Avatar.js

```diff
@@ -1,9 +1,8 @@
-// Avatar component takes person object as props
```

Comment:
Most of the comments here are for business requirements and they could be handy in code base, so you don't need to delete them